modules/common/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/common/config/Config.scala

```diff
@@ -245,8 +240,14 @@ object Config {
   implicit val metricsDecoder: Decoder[Metrics] =
     deriveDecoder[Metrics]
 
-  implicit val opsGenieDecoder: Decoder[OpsGenie] =
-    deriveDecoder[OpsGenie]
+  implicit val http4sUriDecoder: Decoder[Uri] =
+    Decoder[String].emap(s => Either.catchOnly[ParseFailure](Uri.unsafeFromString(s)).leftMap(_.toString))
+
+  implicit val minuteDecoder: Decoder[FiniteDuration] =
+    Decoder[Long].emap(d => Either.catchNonFatal(FiniteDuration(d, MINUTES)).leftMap(_.toString))
```

Comment:
1. It's not a `minute` decoder - it's `FiniteDuration` encoder
2. We see a worst of two worlds here: we're decoding `Long` without a clear indication of what it represents (`Long` usually means milliseconds, not minutes) and at the same time hide this ambiguity with `FiniteDuration`. The `FiniteDuration` must be able to parse typical HOCON `1 minute` or `20 seconds` kinds of terms.